app/models/manageiq/providers/openstack/network_manager/cloud_subnet.rb

```diff
@@ -55,13 +55,12 @@ def with_provider_object
     super(connection_options)
   end
 
-  private
-
   def self.connection_options(cloud_tenant = nil)
     connection_options = {:service => "Network"}
     connection_options[:tenant_name] = cloud_tenant.name if cloud_tenant
     connection_options
   end
+  private_class_method :connection_options
```

Comment:
I checked the other PR (#12041) and I didn't notice changes to this file: app/models/manageiq/providers/openstack/network_manager/cloud_subnet.rb

So, for the moment I keep my changes, let me know if is ok for you.